doc/development/developers_guide.md

```diff
@@ -0,0 +1,66 @@
+# Developers Guide
+
+This page contains information for developers.
+
+## Integration tests
+For the integration tests you can select between three different test environments:
+
+|Setup                          |Description                                |
+|-------------------------------|-------------------------------------------|
+| `testcontainers` (default)    | For each integration test class a separate docker container is started. This method provides best isolation. It is used by the CI tests.|
+|`local`                        | This method is faster that `testcontainers` since it reuses the same container all the time. You must start the container by hand by running `docker-compose up` in `src/test/resources/`| 
+|`aws`                          | This method fits best for performance tests. You have to create the cluster setup first. You can find more information in the [Performance Testing](#performance-testing) section 
+
+You can select the test environment by adding `-Dtests.testSetup="<selected setup>"` as JVM option for the integration tests. 
+
+## Logging & Debugging
+
+The following links explain logging and debugging for the Virtual Schema in general:
+
+* [Logging for Virtual Schemas](https://github.com/exasol/virtual-schemas/blob/master/doc/development/remote_logging.md)
+* [Remote debugging Virtual Schemas](https://github.com/exasol/virtual-schemas/blob/master/doc/development/remote_debugging.md)
+
+For logging and debugging in the test code, things are easier, 
+as most of the setup code is already included in the AbstractExasolTestInterface.
+
+The tests automatically redirect the logs from the Virtual Schema to the tests command line output.
+
+For debugging start a debugger on your development machine listening on port `8000` and 
+start the tests with: `-Dtests.debug="virtualSchema"` or `-Dtests.debug="all"`. 
+The last option also starts the debugger for the UDF calls. This will however typically fail,
+ as the UDFs run in parallel and therefore only one can connect to the debugger.
```

Comment:
Does it always fail with UDF? Or only sometimes?
Maybe we can reword this:
```
This however [usually] fails, as the UDFs run in parallel and therefore only one can connect to the debugger.
```
Add or remove usually if it does not fail always.